README.md

```diff
@@ -48,22 +48,20 @@ Install all dependencies:
 yarn
 ```
 
-Build nebula.js visualization:
+Build a nebula.js visualization as a Qlik Sense extension:
 
 ```sh
 yarn build
 ```
 
-Build a nebula visualization as a Qlik Sense extension:
-
-```sh
-yarn sense
-```
-
 |                          [Saas Edition of Qlik Sense]                           |                     [Qlik Sense Enterprise]                      |                            [Qlik Sense Desktop]                            |
```

Comment:
The desktop example looks weird. It with backslashes so that is a windows example?

Reply:
It is also a bit misleading that this table is under the heading `Steps to create a sn-table extension as as a Qlik Sense extension` since moving the extension doesn't have anything to do with creating it. Change it to `building and adding the sn-table extension to Qlik Sense`

Reply:
Correct
<img width="743" alt="Screenshot 2022-03-07 at 13 08 10" src="https://user-images.githubusercontent.com/4471489/157031771-8be4cb6e-b5a7-49b7-91eb-4c1440c00ad0.png">

Right, gonna update